.github/workflows/build_hub.yml

```diff
@@ -106,7 +91,26 @@ jobs:
           docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite
 
       - name: Load settings
-        run: docker-compose -f docker-compose.yml up settings
+        run: ./gradlew reloadSettings
```

Comment:
Moved the integration tests to run after the container is running and added reloadTables too.